babel.config.json

```diff
@@ -4,19 +4,42 @@
         "@babel/typescript",
         "@babel/react",
         ["@babel/env", { "modules": false }],
-        "@dr.pogodin/babel-preset-svgr"
+        [
+            "@dr.pogodin/babel-preset-svgr",
+            {
+                "svgr": {
+                    "plugins": ["@svgr/plugin-svgo", "@svgr/plugin-jsx", "@svgr/plugin-prettier"],
+                    "svgoConfig": {
+                        "plugins": [
+                            {
+                                "name": "preset-default",
+                                "params": {
+                                    "overrides": {
+                                        "removeViewBox": false,
+                                        "cleanupIds": false
+                                    }
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
```

Comment:
This took me a while to understand how to properly deactivate cleaning up ids plugin of SVGO, if you're  interested to know why is this needed, please see: https://github.com/prezly/slate/pull/477#discussion_r1304336589